src/components/search/map/marker.tsx

```diff
@@ -56,8 +58,25 @@ export const Marker = ({
     marker.setFocusHandler((focused: boolean) => onFocus(focused, id));
     marker.setHoverHandler((hovered: boolean) => onHover(hovered, id));
 
+    // Add the pin to the pinStore if it is not a cluster marker
+    const isClusterMarker = id.includes("cluster-");
```

Comment:
Since the ids are normally the entity ids, I think there's a rare chance that we get a false positive here. Is there a different way we could do this?